api/src/test/java/com/epam/pipeline/manager/docker/DockerContainerOperationManagerTest.java

```diff
@@ -167,7 +179,8 @@ public void runShouldBeStillRunningIfCommandCannotExecute() throws IOException,
         when(sshConnection.waitFor(anyLong(), any())).thenReturn(true);
         when(sshConnection.exitValue()).thenReturn(CANNOT_EXECUTE_EXIT_CODE);
         final PipelineRun run = pipelineRun();
-
+        doReturn(null).when(preferenceManager).getPreference(SystemPreferences.BASE_API_HOST);
```

Comment:
Why do we need this mock?

Reply:
We use this mock in the `launchPauseRunScript` method in the following line 

`final String pauseRunCommand = DockerPauseCommand.builder()
                .runPauseScriptUrl(pauseRunScriptUrl)
                .api(preferenceManager.getPreference(SystemPreferences.BASE_API_HOST))`

Reply:
Right, but why do we need to mock this method? (`preferencemanager` returns `null` for each method excepting mocked)

Reply:
Yes, you're right. It's not necessary mock.